ios/App/Shared/player/PlayerHandler.swift

```diff
@@ -31,15 +28,11 @@ class PlayerHandler {
         player = AudioPlayer(sessionId: sessionId, playWhenReady: playWhenReady, playbackRate: playbackRate)
     }
     
-    public static func stopPlayback() {
+    public static func stopPlayback(currentSessionId: String? = nil) {
```

Comment:
New parameter ensures the current playback session does not get invalidated during restore, which would cause it not be restored the next time.